src/main/resources/com/google/api/codegen/java/disco_schema.snip

```diff
@@ -0,0 +1,141 @@
+@extends "java/common.snip"
+@extends "java/method_sample.snip"
+
+@snippet generate(apiFile)
+  {@renderFileHeader(apiFile.fileHeader)}
+
+  @@Generated("by GAPIC")
+  @@BetaApi
+  public final class {@apiFile.schema.typeName} implements Serializable {
+    {@members(apiFile.schema)}
+    {@constructor(apiFile.schema)}
+    {@getters(apiFile.schema)}
+    {@newBuilder(apiFile.schema)}
+    {@builder(apiFile.schema)}
+    {@toString(apiFile.schema)}
+    {@equalsMethod(apiFile.schema)}
+    {@hashCode(apiFile.schema)}
+  }
+@end
+
+@private toString(xschema)
+  @@Override
+    public String toString() {
+      return "{@xschema.typeName}{"
+      @join property : xschema.properties
+          + "{@property.name}=" + {@property.name} + ", "
```

Comment:
Changing the join to joining on the string ", " so there is no longer a trailing comma.